open-craft/problem-builder
# SE-1913 python3 compatibility

This PR adds python3 compatibility (testing with python3.5), while retaining support for python2.7. 

**Test instructions**:

- ensure all tests pass
- test manually on a devstack with at least python2 to sanity check

**Author notes/concerns**:



**Reviewers**:
